### PR TITLE
files referring to ns1labs

### PR DESCRIPTION
**Pull request by manrodrigues**
Ready to merge

### CODE REVIEW

File: .github/workflows/code-ql.yml [hunk 1]
```diff
@@ -68,7 +68,7 @@ jobs:
         # create conan config
         CONAN_V2_MODE=1 conan config init
         conan config set general.revisions_enabled=1
-        conan remote add ns1labs https://ns1labs.jfrog.io/artifactory/api/conan/ns1labs-conan
+        conan remote add orbcommunity https://orbcommunity.jfrog.io/artifactory/api/conan/orbcommunity-conan
```
Comment (manrodrigues) [inline .github/workflows/code-ql.yml]: @etaques plase confirm if it must be orbcommunty or orb-communty

File: .github/workflows/debug_build.yml [hunk 2]
```diff
@@ -190,7 +190,7 @@ jobs:
 
       - name: Build, push debug image + crashhandler
         env:
-          IMAGE_NAME: ns1labs/pktvisor
+          IMAGE_NAME: orbcommunity/pktvisor
```
Comment (manrodrigues) [inline .github/workflows/debug_build.yml:193]: @etaques is this two images necessary? both are the same

File: README.md [hunk 3]
```diff
@@ -524,14 +524,14 @@ You can set the `instance` label by passing `--prom-instance ID`
...
-cloud providers, there is a [docker image available](https://hub.docker.com/r/ns1labs/pktvisor-prom-write) to make this
+cloud providers, there is a [docker image available](https://hub.docker.com/r/orbcommunity/pktvisor-prom-write) to make this
 easy. See [centralized_collection/prometheus](centralized_collection/prometheus) for more.
 
 Also see [getorb.io](https://getorb.io) for information on connecting pktvisor agents to the Orb observability platform.
 
 ### REST API
 
-REST API documentation is available in [OpenAPI Format](https://app.swaggerhub.com/apis/ns1labs/pktvisor/3.0.0-oas3)
+REST API documentation is available in [OpenAPI Format](https://app.swaggerhub.com/apis/orbcommunity/pktvisor/3.0.0-oas3)
```
Comment (manrodrigues) [inline README.md]: @etaques is this link correct? i'm not able to access it

File: .github/workflows/code-ql.yml (hunk 1 above)
Comment (etaques) [reply .github/workflows/code-ql.yml]: seems that jfrog is not being used more

File: README.md (hunk 3 above)
Comment (etaques) [reply README.md]: use **orb-community** instead **orbcommunity**

**Review by etaques — DISMISSED** [review]
LGTM

**Review by etaques — APPROVED** [review]
LGTM

**Pull request merged**